src/block/icon/style.js

```diff
@@ -92,6 +93,23 @@ export const createStyles = props => {
 		styles.push( ...createResponsiveStyles( '.ugb-icon__title', 'title%sBottomMargin', 'marginBottom', '%spx', props.attributes, true ) )
 	}
 
+	// Advanced Column Gap
+	styles.push( {
+		'.ugb-icon__content-wrapper': {
+			columnGap: appendImportant( getValue( 'columnGap', '%spx' ) ),
+		},
+		tablet: {
+			'.ugb-icon__content-wrapper': {
+				columnGap: appendImportant( getValue( 'tabletColumnGap', '%spx' ) ),
+			},
+		},
+		mobile: {
+			'.ugb-icon__content-wrapper': {
+				columnGap: appendImportant( getValue( 'mobileColumnGap', '%spx' ) ),
+			},
+		},
+	} )
+
```

Comment:
You can use `createResponsiveStyles` here to compress this whole thing into just 1 line.